demos/showcase/src/App.js

```diff
@@ -107,14 +109,20 @@ export default () => {
                                     title: titleList[0],
                                     itemID: titleList[0],
                                     icon: <ListIcon />,
-                                    onClick: () => setRoute(0),
+                                    onClick: () => {
+                                        setRoute(0);
+                                        if (xsDown) setOpen(false);
+                                    },
                                 },
                                 {
                                     title: titleList[1],
                                     itemID: titleList[1],
                                     subtitle: '2 Alarms',
                                     icon: <NotificationsActive />,
-                                    onClick: () => setRoute(1),
+                                    onClick: () => {
+                                        setRoute(1);
+                                        if (xsDown) setOpen(false);
```

Comment:
do we need to do this for all of them, or should we do it even for the persistent drawer?

Reply:
I think the persistent drawer is already auto collapsing itself (through the onItemSelect function).